tools/wake/main.cpp

```diff
@@ -190,6 +191,14 @@ class TerminalReporter : public DiagnosticReporter {
 };
 
 int main(int argc, char **argv) {
+  // Make sure we always get core dumps but don't fail
+  // if that fails for some reason.
+  struct rlimit core_lim;
+  getrlimit(RLIMIT_CORE, &core_lim);
```

Comment:
I'm wondering if we should restore the original limit after we next `fork()` (where ever that might be), as here we enable core dumps for all payload commands run inside wakebox

Reply:
I'm not personally very apposed to getting core dumps for everything but if people want that I can make shim-wake reset the limit or reset it just before invoking shim wake.